kinoml/modeling/homology.py

```diff
@@ -0,0 +1,272 @@
+from ..core.proteins import ProteinStructure
+from ..core.sequences import KinaseDomainAminoAcidSequence
+
+
+class HomologyModel:  #  TODO inherent a Base class?
```

Comment:
We could devise a `ModellingMethod` class that provides some utilities if needed.